ddtrace/internal/_encoding.pyx

```diff
@@ -585,7 +585,7 @@ cdef class MsgpackEncoderV03(MsgpackEncoderBase):
                 if ret != 0:
                     return ret
                 # pack the value of a span link field (values can be number, string or dict)
-                if isinstance(v, Number):
+                if isinstance(v, NumericType):
```

Comment:
```suggestion
                if isinstance(v, (int, float)):
```

This is all the compat is doing, and since we aren't trying to support Python 2.7 we don't really need to use `six` (IMHO)